docs/howto/account-billing/Delete-account.md

```diff
@@ -0,0 +1,19 @@
+# Delete account
+
+If you unfortunately need to cancel or leave us for any reason, you can delete your account in our [{{gui}}](https://{{gui_domain}}).
+Log in and click on the Profile button at the top right.
+
+![Orange avatar person with white
+background](assets/picture-profile.png)
+
+You will now see the _Account settings_ page. Choose the _Settings_ tab, in which you can delete your account.
+
+![A Settings tab with a red box around it](assets/accountsettings.png)
+
+Scroll to the bottom until you see the image below and if you absolutely sure click on the red button _Delete Account_ and follow the procedure.
+
+![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)
+
+Your services will be shut off in 3 days and from now your services stop costing you money.
+
+If you regret what you just did, please contact us directly at support@cleura.com and we will do everything in our power to save your account.
```

Comment:
These sound awkward and will need rephrasing. I'll leave it to @colder-is-better to provide suggestions. :)

Reply:
It´s me who wanna save any customer at any price :-)

Reply:
But colder-is-better wrote it better